### ISSUE TITLE
Add support django-mptt

### ISSUE THREAD

**Issue by vage-mel** [unfoldadmin/django-unfold#366]
Can you add support django mptt?

**Comment by lukasvinclav** [comment]
It seems that the project django-mptt is not maintained anymore. Are you still using it?

**Comment by vage-mel** [comment]
Yes, we use

**Comment by vage-mel** [comment]
@lukasvinclav can you add support django mptt it's very important for us

**Comment by lukasvinclav** [comment]
I just checked the mptt project page again, and the project is not maintaned anymore. If you really need mptt in Unfold you can still create your own implementation.

**Comment by vasilistotskas** [comment]
Is there any workaround on usage along with django mptt ?

**Comment by Zerotask** [comment]
> Is there any workaround on usage along with django mptt ?

We're facing the same issue. We noticed that the breaking change was added in v0.51.0. The latest working version was v0.50.0

So one (temporary) workaround would be using v0.50.0.
Another solution would be to identify the specific breaking changes (https://github.com/unfoldadmin/django-unfold/compare/0.50.0...0.51.0) and override the parts. This will eventually cause other issues when new versions of django-unfold are released.

Probably the best solution would be to rewrite your own code and migrate from django-mptt to another (maintained) package and hope this will have a better support in django-unfold.
One thing to notice that even if mptt's README says it's unmaintained, the last commit was 3 months ago.
Probably the best alternative would be [django-treebeard](https://django-treebeard.readthedocs.io/en/latest/admin.html) which also uses nested sets, have support for Django Admin and the GitHub relevant KPIs look better: last commit a month ago, 25 watching, 218 forks, 1.2k stars, 71 contributors
django-mptt also mentions [django-tree-queries](https://github.com/feincms/django-tree-queries) which does not support nested sets but adjacency lists and CTEs. The GitHub KPIs are okay, nut as good as django-treebeard. It also doesn't seem to support Django Admin (UI) stuff out of the box.
In our case, we will probably look into django-treebeard
Update: https://github.com/fabiocaccamo/django-treenode could also be an alternative to look into.
